### ISSUE TITLE
Import creates seconds packages folder and does not stage new package folder

### ISSUE THREAD

**Issue by simondel** [lerna/lerna#767]
I imported a package using command: `lerna import C:\temp\stryker-mocha-runner` and all my commits were applied as I had expected. When I pushed the branch to github I noticed that it had created a separate folder which does not exist on my disk. However, Git still thinks the folder is untracked on my local Git repo.

## Expected Behavior
`lerna import C:\temp\stryker-mocha-runner` imports the specified package and replays every commit (when you ask for it). After lerna is done, there are 0 untracked files on my local disk and GitHub thinks there is only one `packages` folder.

## Current Behavior
If you look at [this branch](https://github.com/stryker-mutator/stryker/tree/d45a9e7412bcdae72dca2336e3e0f5282d83824d) you see there is a `packages` folder and a `packages\stryker-mocha-runner` folder:
![image](https://cloud.githubusercontent.com/assets/4221997/25061082/5827efdc-21ae-11e7-8fb9-917842b24919.png)

On my pc:
![image](https://cloud.githubusercontent.com/assets/4221997/25061038/5d8d76e6-21ad-11e7-9fb7-e43f03c54e14.png)

## Possible Solution
None so far

## Steps to Reproduce (for bugs)
1. Clone the stryker repo on commit [5dd4eaf549](https://github.com/stryker-mutator/stryker/tree/5dd4eaf549c31f2153ebcd55403c431cbd3d0136)
2. Clone the [stryker-mocha-runner repo](https://github.com/stryker-mutator/stryker-mocha-runner)
3. Import the stryker-mocha-runner repo and specify you want to replay the commits
4. Push the current branch to GitHub

<details><summary>lerna.json</summary><p>

```json
{
  "lerna": "2.0.0-rc.0",
  "packages": [
    "packages/*"
  ],
  "version": "independent"
}
```
</p></details>

## Context
I want to import a package without losing history

## Your Environment
| Executable | Version |
| ---: | :--- |
| `lerna --version` | 2.0.0-rc.2|
| `npm --version`  | 3.10.10|
| `yarn --version` | 0.17.9 (I don't use yarn)|
| `node --version` | 6.9.5|

| OS | Version |
| --- | --- |
| Windows 10 Enterprise| 1607|

**Comment by simondel** [comment]
I just commited the untracked folder and pushed it to see what happened. Now stryker-mocha-runner exists twice in Github and still once on disk:
![image](https://cloud.githubusercontent.com/assets/4221997/25061117/0e116e72-21af-11e7-9585-e466da5c9970.png)

And if I click the `packages\stryker-mocha-runner` folder in the root of the repo:
![image](https://cloud.githubusercontent.com/assets/4221997/25061115/fb2263de-21ae-11e7-968d-b12da86e053b.png)

**Comment by evocateur** [comment]
This looks like a path corruption problem during the patch mangling during the import execution. The regular expressions aren't handling Windows path separators correctly, if I had to guess.

**Comment by unional** [comment]
Confirm this bug still exist on 2.0.0 release version

**Comment by Adam13531** [comment]
I found a workaround for this issue. The core of the problem is that `path.join` in ImportCommand.js will concatenate paths with a backslash on Windows ([source code here](https://github.com/lerna/lerna/blob/master/src/commands/ImportCommand.js#L74)).

To quickly/hackily fix this for myself, I added this line of code to `lib/commands/ImportCommand.js` directly after line 125 (note: this is transpiled code):
```js
this.targetDir = _path2.default.join(targetBase, externalRepoBase); // already on line 125
this.targetDir = this.targetDir.replace(/\\/g, '/'); // I added this
```

If either of the paths naturally had a backslash in them, then this workaround will break.
